odpf/optimus/core/v1beta1/job_spec.proto

```diff
@@ -276,7 +279,23 @@ message RefreshJobsResponse {
 
   // response type differentiate events
   string type = 2;
-  string message = 3;
+  string value = 3;
 
   string job_name = 4;
 }
+
+message GetDeployJobsStatusRequest {
+  string deploy_id = 1;
+}
+
+message GetDeployJobsStatusResponse {
+  string status = 1;
+
+  repeated DeployJobFailure failures = 2;
+  int32 total_succeed = 3;
```

Comment:
wouldn't it be better to have the failed as well as the total jobs.

Reply:
lets have: success_count, failure_count